_includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-offsets.adoc

```diff
@@ -0,0 +1,22 @@
+So far you've learned how to consume records from a specific partition. When you specify the partition, you can optionally specify the offset to start consuming from.  Specifying a specific offset can be helpful when debugging an issue, in that you can skip consuming records that you know aren't a potential problem.
+
+If you haven't done so already, close the previous console consumer with a `CTRL+C`.
+
+From the previous step you know there are 6 records in the second partition.  In this step you'll only consume records starting from offset 3, so you should only see the last 3 records on the screen. The changes in this command include removing the `--from-beginning` property and adding an `--offset` flag
+
+
+Here's the command to read records from the second partition starting at offset 6:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/console-consumer-read-specific-offsets-partition/confluent/code/tutorial-steps/dev/console-consumer-keys-partition-offset.sh %}</code></pre>
++++++
+
+After a few seconds you should see something like this
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/console-consumer-read-specific-offsets-partition/confluent/code/tutorial-steps/dev/expected-output-step-three.txt %}</code></pre>
++++++
+
+So you can see here, you've consumed records starting from offset 3 to the end, which includes record with offsets of `3, 4, and 5` the last three records.
```

Comment:
```suggestion
So you can see here, you've consumed records starting from offset `3` to the end, which includes records with offsets of `3`, `4`, and `5` (the last three records).
```
I can see this is language carried over from the original. Maybe it's a me problem but I'm having a hard time understanding what this is conveying. I'm not sure if `and` needs to be in code. Feel free to disregard if you think the original is fine as is!

Reply:
@awalther28 agree, it's confusing.  I've wordsmith'd both versions in latest commit.